### PR TITLE
add date_to_weekday finder method

### PR DESCRIPTION
**Pull request by suryapratapsinghsuryavanshi**
### **Describe your change:**

* [x] Add an algorithm?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: other/date_to_weekday.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        "Wednesday",
+        "Thursday",
+        "Friday",
+        "Saturday",
+        "Sunday",
+    ]
+    day, month, year = [int(x) for x in inp_date.split("/")]
+    if year % 100 == 0:
+        year = "00"
+    new_base_date: str = f"{day}/{month}/{year%100} 23:15:59"
```
Comment (L3str4nge) [inline other/date_to_weekday.py:28]: I think we don't need a time here.

File: other/date_to_weekday.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    :param inp_date:
+    :return: String
+    >>> date_to_weekday("7/8/2035")
+    'Tuesday'
+    >>> date_to_weekday("7/8/2021")
+    'Saturday'
+    >>> date_to_weekday("1/1/2021")
+    'Friday'
+    """
+    day_list: list = [
```
Comment (L3str4nge) [inline other/date_to_weekday.py:16]: We have pretty module in python so we don't need to hardcode weekdays `import calendar;calendar.day_name`

**Review by L3str4nge — CHANGES_REQUESTED** [review]

**Review by L3str4nge — CHANGES_REQUESTED** [review]
Tests are failing

**Comment by suryapratapsinghsuryavanshi** [comment]
Yes, but I don't get a reason why the tests are failing.

**Review by L3str4nge — APPROVED** [review]

**Pull request merged**